tests/unit/kats/hybrid_ecdhe_sike_r1.kat

```diff
@@ -34,7 +34,7 @@ Point data:                          04CA42DFDCFF6D9FC5CA5BC7ECCDCDAFDB49BB5E91E
 SIKE CT len:       402 bytes         0192
 SIKE CT:                             AF3F754332333692E8C5BA533A146F2A5C155D2E7D251D7729B26ACB62DC4D201C82DE4A0B1E091BCA03A0D73AFD858EFBF4E9EA9E9C6278B4BEE10B54323C9B67994B5A88D066A84A6AC5D9B8572F2FF762CF5F501E4AACB9291EDFBAE2DEEB094AA2A2B2965B72701D69850578457FDB5510BA7790AAC84E46B181CF3167CA49E6C088208470AA05D00A03E1C54820A3F9A69E0AB897556040699F080E48D9E95FCAA4DC5D9B86F21D715CC1498AEBC5A978E092F0078C04A93DD13CECE1B52F687AA7F1B2177B3AD2B052C572087E4C989B7652DDA16778152596ECDA4F377D079EA84337CDD31B4EB700FF0388056DDC48F5CE4283E09908ED330DCDB6D199ADA83F6AFBEA50B41A4F589A308AE920268D1AB826E294FDA00B8E1EE81BDCD1BB8EF72371B89569333ABD3DC808DD856DD23C2820BD41B7F00ED481B8DE1D2887FBA3C2E1B863ADEA21A4BD67A8F05E7200D0B1D648B29B32502378A5E6F8C5933DC838CAC1AD73944605F189D74E797ED6376257723A2B2D8A2D3F276D6003F677EB3CC75B5A15FA181B8A8A410F933B
 
-seed = 00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
+seed = 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```

Comment:
Why are we updating a Round 1 KAT file? Shouldn't this stay the same to ensure backwards compatibility?

Reply:
The actual content of the KAT is unchanged. We only read 48 bytes (96 hex characters) from the seed value (see https://github.com/awslabs/s2n/blob/master/tests/testlib/s2n_hybrid_kem_tests.c#L119). For some reason, this KAT seed was written with 64 bytes (128 hex characters). The extras just get ignored, so I updated it to be consistent with all the other KATs.

If the actual seed value had changed meaningfully, then the client/server messages and shared secret would have changed.